browser/ui/views/omnibox/brave_search_conversion_promotion_view.cc

```diff
@@ -464,28 +463,29 @@ void BraveSearchConversionPromotionView::ConfigureForButtonType() {
           .WithOrder(2));
 
   auto* try_button = button_type_container_->AddChildView(
-      std::make_unique<TryButton>(views::Button::PressedCallback(
+      std::make_unique<CustomMdTextButton>(views::Button::PressedCallback(
           base::BindRepeating(&BraveSearchConversionPromotionView::OpenMatch,
                               base::Unretained(this)))));
+  try_button->SetKind(views::MdTextButton::Kind::kPrimary);
   try_button->SetProperty(views::kMarginsKey, gfx::Insets::VH(17, 0));
-  try_button->SetProminent(true);
   try_button->SetText(brave_l10n::GetLocalizedResourceUTF16String(
       IDS_BRAVE_SEARCH_CONVERSION_TRY_BUTTON_LABEL));
   try_button->SetFontSize(12);
 
-  auto* dismiss_button = button_type_container_->AddChildView(
-      std::make_unique<ButtonTypeDismissButton>(views::Button::PressedCallback(
+  auto* close_button = button_type_container_->AddChildView(
+      std::make_unique<CloseButton>(views::Button::PressedCallback(
           base::BindRepeating(&BraveSearchConversionPromotionView::Dismiss,
                               base::Unretained(this)))));
+  constexpr int kButtonTypeCloseButtonSize = 16;
   views::SetImageFromVectorIconWithColor(
-      dismiss_button, vector_icons::kCloseRoundedIcon,
-      GetLayoutConstant(LOCATION_BAR_ICON_SIZE),
-      SkColorSetRGB(0x6B, 0x70, 0x84), gfx::kPlaceholderColor);
-  views::InstallCircleHighlightPathGenerator(dismiss_button);
-  views::FocusRing::Install(dismiss_button);
-  dismiss_button->SetProperty(views::kMarginsKey, gfx::Insets::VH(25, 8));
-  dismiss_button->SetTooltipText(brave_l10n::GetLocalizedResourceUTF16String(
-      IDS_BRAVE_SEARCH_CONVERSION_DISMISS_BUTTON_TOOLTIP));
+      close_button, kLeoCloseIcon, kButtonTypeCloseButtonSize,
+      // Use Semantic/Icon/Default when it's available from leo/colors.h.
+      SkColorSetRGB(0x69, 0x72, 0x84), gfx::kPlaceholderColor);
```

Comment:
Could be related? https://github.com/brave/leo/pull/313#issuecomment-1652784980

cc. @petemill

Reply:
Note: for now, we don't have the `Semantic` or `Themable` segment in our paths, so you should be able to use the `kColorIconDefault` token here :smile:

Reply:
From the figma, it's `#687485` but `kColorIconDefault` from leo/colors.h is different hex value with it.
What should I do?
<img width="247" alt="Screenshot 2023-07-27 at 1 57 18 PM" src="https://github.com/brave/brave-core/assets/6786187/6d8bef89-0435-4975-a932-30a7637853f1">

Reply:
Maybe the leo/color.h isn't just updated yet. I think I've seen some values are out of sync with the latest design according to leo update schedule 😄

Reply:
Ah, got it. Updated to use `kColorIconDefault` 👍🏼

Reply:
Yeah, @aguscruiz is going through a massive refactor, and he's tweaked a few token values too - once https://github.com/brave/brave-core/pull/19448 lands the value should match (but it's probably going to be a few days at least).